src/poprox_recommender/handler.py

```diff
@@ -16,29 +17,23 @@ def load_model(device_name=None):
     if device_name is None:
         device_name = "cuda" if th.cuda.is_available() else "cpu"
 
-    load_path = f"{project_root()}/models/ckpt-30000.pth"
-    with smart_open(load_path, "rb") as f:
-        buffer = io.BytesIO(f.read())
+    load_path = f"{project_root()}/models/model.safetensors"
+    #with smart_open(load_path, "rb") as f:
+        #buffer = io.BytesIO(f.read())
 
-    if buffer:
-        device = th.device(device_name)
-        checkpoint = th.load(buffer, map_location=device)
+    #if buffer:
+        #device = th.device(device_name)
+        #checkpoint = th.load(buffer, map_location=device)
```

Comment:
If we're not using these commented out lines, let's remove them—if we ever needed to recover them for some reason, we could find them in the commit history.